### DISCUSSION TITLE
Does Theia support VSCode Web extensions?

### DISCUSSION THREAD

**Discussion by Ark-kun** [eclipse-theia/theia#11468] [Q&A]
VSCode Web was released with VSCode 1.62 while Theia is at 1.53.

I've recently published my [Pipeline Editor] custom editor extension to open-vsx registry https://open-vsx.org/extension/Cloud-pipelines/pipeline-editor-vscode , but I do not see the extension in Theia search.

More importantly, when I manually install the extension from vsix, it's registered, but when I try to open a file in the Pipeline Editor, an editor tab does not open. I do not see any issue sin the console (other than that menu/newFile does not exist).

**Comment by vince-fugnitto** [comment]
@Ark-kun thank you for the discussion, for the most part they should in fact work thanks to https://github.com/eclipse-theia/theia/pull/10721.

> I've recently published my [Pipeline Editor] custom editor extension to open-vsx registry https://open-vsx.org/extension/Cloud-pipelines/pipeline-editor-vscode , but I do not see the extension in Theia search.

I can't confirm for sure since the repository link to your extension actually points to [pipeline-editor](https://github.com/Cloud-Pipelines/pipeline-editor) but I believe it is for the simple fact we only display extensions which are compatible given our API version. If you specified an API (using `engines.vscode`) of `1.62` then it will not show up in the search since there is a greater chance of incompatibilities. 

> More importantly, when I manually install the extension from vsix, it's registered, but when I try to open a file in the Pipeline Editor, an editor tab does not open. I do not see any issue sin the console (other than that menu/newFile does not exist).

Unfortunately iIt is difficult to diagnose for us without actually seeing the sources.
